lib/Onyx.js

```diff
@@ -35,6 +35,42 @@ let defaultKeyStates = {};
 // Connections can be made before `Onyx.init`. They would wait for this task before resolving
 const deferredInitTask = createDeferredTask();
 
+/**
+ * Returns a simplifiedVersion of the object by using the selector
+ * @param {Object} sourceObject
+ * @param {String|Function} selector
+ *      If it's a string, the selector is passed to lodashGet on the sourceObject
+ *      If it's a function, it is passed the sourceObject and it should return the simplified object
+ * @returns {Object}
+ */
+const getSimplifiedObject = (sourceObject, selector) => (_.isFunction(selector)
```

Comment:
No, it definitely doesn't HAVE to return an object. Maybe I should clarify that it can return `mixed`. It doesn't really matter what is returned as long as the developer implementing it knows what to expect and codes everything else to handle whatever value they return.